tests/ecmascript/test-bi-math-cbrt.js

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Math.cbrt()
+ */
+
+/*@include util-number.js@*/
+
+/*===
+function true false true
+1
+0
+-0
+Infinity
+-Infinity
+NaN
+2
+4
+10
+-2
+2.5
+-2.5
+812
+===*/
+
+function test() {
+    var pd = Object.getOwnPropertyDescriptor(Math, 'cbrt');
+    print(typeof pd.value, pd.writable, pd.enumerable, pd.configurable);
+    print(Math.cbrt.length);
+
+    printExact(Math.cbrt(0));
+    printExact(Math.cbrt(-0));
+    printExact(Math.cbrt(Infinity));
```

Comment:
This is fine but generally testcases use `1/0`, `-1/0`, and `0/0` for +Inf, -Inf, and NaN. Not important as such, just avoids a dependency on the global binding.